tasks/sysctl-setup.yml

```diff
@@ -8,14 +8,40 @@
     or ansible_distribution_major_version | int < 10
 
 # See: https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#letting-iptables-see-bridged-traffic
-- name: Let iptables see bridged traffic.
-  sysctl:
```

Comment:
Is there some reason the `sysctl` module can't be used for these modifications (and `lineinfile` used instead)?

Reply:
I do not remember.

I've just complied with https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#letting-iptables-see-bridged-traffic .

Reply:
This is immediately applied, but will only last 'till the next boot, so it is not a permanent change.
to make sysctl changes permanently write `token = value` formatted file into below dirs.

```
FILES
       /run/sysctl.d/*.conf
       /etc/sysctl.d/*.conf
       /usr/local/lib/sysctl.d/*.conf
       /usr/lib/sysctl.d/*.conf
       /lib/sysctl.d/*.conf
       /etc/sysctl.conf
              The paths where sysctl preload files usually exist.  See also sysctl option --system.
```